konflux-configs/base/project/overlay/rekor-monitor/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - project.yaml
+  - template.yaml
+
+components:
+  - ../../base/ec
+  - ../../base/monorepo
+  - ../../base/release-plan
```

Comment:
Generated ReleasePlan plan template has incorrect value in `spec.tenantPipeline.params.[name=type].value`. It should be ``component` currently it is set for `operator`.

I think that solution will be to modify `release-plan` trait to decide based on `build.rhtas.com/type` label value. You can find existing solution in `ec` trait.

Reply:
other examples are `ocp/v4.x/` they uses that type label to apply only for fbc type